apps/davi/src/stores/types.ts

```diff
@@ -153,6 +153,21 @@ export interface FetcherHooksInterface {
     status: ProposalState,
     endTime: Moment
   ) => { detail: string; moment: Moment };
+  useGetVotingMachineAddressBySchemeAddress: (schemeAddress: string) => {
+    votingMachineAddress: string;
+    isError: boolean;
+    isLoading: boolean;
+  };
+  useVotingPowerOfAt: (
+    contractAddress: string,
+    userAddress: `0x${string}`,
+    snapshotId?: string,
+    fallbackSnapshotId?: boolean
+  ) => {
+    data: BigNumber;
+    isError: boolean;
+    isLoading: boolean;
+  };

```

Comment:
Since we're not using `useVotingPowerOfAt` in the components but on the hooks, we can skip adding it to the hook store.

Currently, we also import it directly from the file, so nothing will be affected.